### PR TITLE
Chessboard

### PR DESCRIPTION
**Pull request by waywho**
Created chess board (back to ruby land!) using board methods in game.rb. Added image values to populate_board method. Also hardwired the games/:id route for now in routes table. Used the populate_board to initialize the board, then switched the code to existing board.

### CODE REVIEW

File: app/models/game.rb [hunk 1]
```diff
@@ -29,40 +29,40 @@ def populate_board
...
+    @board[6][0] = Pawn.create(:color => :black, :name => :pawn, :position => "06", :x_axis => 0, :y_axis => 6, :image => 'black-pawn.gif')
+    @board[6][1] = Pawn.create(:color => :black, :name => :pawn, :position => "16", :x_axis => 1, :y_axis => 6, :image => 'black-pawn.gif')
+    @board[6][2] = Pawn.create(:color => :black, :name => :pawn, :position => "26", :x_axis => 2, :y_axis => 6, :image => 'black-pawn.gif')
+    @board[6][3] = Pawn.create(:color => :black, :name => :pawn, :position => "36", :x_axis => 3, :y_axis => 6, :image => 'black-pawn.gif')
+    @board[6][4] = Pawn.create(:color => :black, :name => :pawn, :position => "46", :x_axis => 4, :y_axis => 6, :image => 'black-pawn.gif')
+    @board[6][5] = Pawn.create(:color => :black, :name => :pawn, :position => "56", :x_axis => 5, :y_axis => 6, :image => 'black-pawn.gif')
+    @board[6][6] = Pawn.create(:color => :black, :name => :pawn, :position => "66", :x_axis => 6, :y_axis => 6, :image => 'black-pawn.gif')
+    @board[6][7] = Pawn.create(:color => :black, :name => :pawn, :position => "76", :x_axis => 7, :y_axis => 6, :image => 'black-pawn.gif')
```
Comment (kenmazaika) [inline app/models/game.rb]: This looks like it's missing a database migration for the `:image` field.

File: app/controllers/games_controller.rb [hunk 2]
```diff
@@ -1,2 +1,12 @@
 class GamesController < ApplicationController
+
+	def show
+	@game = Game.find(params[:id])
+	
+	# used to initialize board
+    # @board = @game.populate_board
+
+    @board = @game.board
```
Comment (kenmazaika) [inline app/controllers/games_controller.rb]: Can you fix the indentation here?

**Comment by kenmazaika** [comment]
Solid work, WeiHsi! :)

File: app/models/game.rb (hunk 1 above)
Comment (waywho) [inline app/models/game.rb]: sorry I am not sure what you mean. The :image field exists within the pieces database already in my schema. I think that was already there when the database was created before.

Comment (kenmazaika) [inline app/models/game.rb]: Whoops, you're right.  I'm wrong! :)

**Comment by denismartens** [comment]
I've reviewed this code and am ready to merge! One minor detail I see is a typo on line 5 in show.html.erb - I think that should say "class". I really like the use of nth-child selector, I didn't know such a feature existed.

**Comment by kenmazaika** [comment]
@waywho, can you resolve the merge conflict so we can get this in there?

**Comment by waywho** [comment]
resolved the merge conflicts. adding the :game_id => self.id that was merged in to the master and keeping the :image => field populated with chess piece images. Please merge me in. Thank you!

**Comment by kenmazaika** [comment]
I love seeing these get merged in! :)

**Comment by Neilda** [comment]
I’m running into an interesting problem. I just finished up the indextest and pushed it to GitHub. It was successful on my terminal, but when I went to GitHub, it didn’t show up for me to submit the pull request. I’m not sure what happened. Did anyone else have this problem?

—

**Comment by kenmazaika** [comment]
If it doesn't show up to build a new pull request from the homepage you can
always press this button:

http://take.ms/hHpA1

**Comment by denismartens** [comment]
I haven't had this problem. Did you do: 
git push -u upstream "your-branch-name"?

**Comment by Neilda** [comment]
I’ve never used upstream. I used git push origin indextest2. That method worked for me before and I see the pull request button on GitHub, but it’s not working now. :-/

—

**Comment by kenmazaika** [comment]
@denismartens you used a slightly different method than we generally tell people to do.  You used a fork: https://github.com/denismartens/GreenMonkeyChess.  Since you have direct push access to the repo, you don't need a fork technically, although you're free to use one.  The nice thing about using a fork is your fork is safe from other people messing with.

On the flip side, that's why most people haven't used `push upstream`.

@Neilda if you:

Click here: http://take.ms/Tkwra
And then click here: http://take.ms/9yy1t
You can create a pull request: http://take.ms/9MGAp

Cheers,
Ken

**Pull request merged**